beamer/agent/chain.py

```diff
@@ -522,6 +538,13 @@ def _timestamp_is_l1_finalized(
     return int(time.time()) > timestamp + target_chain_finality
 
 
+def get_l1_cost(context: Context) -> int:
```

Comment:
in `_l1_resolution_threshold_reached` there is still a code duplicate of this. Can you please reuse get_l1_cost there?